.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# IntelliJ configurations
```

Comment:
It seems this file was added by mistake, could you please remove it? Thanks

Reply:
Sure np. But it would be nice if the repo had global `.gitignore` because it is quite annoying to work w/o it when using an IDE.